services/administration/src/EShopOnAbp.AdministrationService.Application/AdministrationServiceApplicationModule.cs

```diff
@@ -11,7 +10,6 @@ namespace EShopOnAbp.AdministrationService
         typeof(AdministrationServiceDomainModule),
         typeof(AdministrationServiceApplicationContractsModule),
         typeof(AbpPermissionManagementApplicationModule),
-        typeof(AbpFeatureManagementApplicationModule),
         typeof(AbpSettingManagementApplicationModule)
```

Comment:
Shouldn't we add `AbpAutoMapperModule` dependency here since you added a reference to the `Volo.Abp.AutoMapper` package?

Reply:
Thank you, added with new commit.